### DISCUSSION TITLE
Is USB HDD supported in HDD db?

### DISCUSSION THREAD

**Discussion by Eltombek** [007revad/Synology_HDD_db#260] [Q&A] [answered]
I would like to connect a USB HDD to my Synology NAS and use it for Hyper Backup and I'm wondering if after using your script I am still limited to compatible drives selection only or it has been sorted by the script as well? Can't find an answer reading in readme or Todo.

Awesome work with HDD db script! It helped me avoid buying new compatible drives as Synology support could not help with drives access error after the latest database update seeing my incompatible drives in use . You deserve some donation.

**Comment by 007revad** [comment]
Any USB HDD is compatible with DSM and can be used by Hyper Backup as a backup location.

**Reply by Eltombek** [reply]
Do you mean after using your script all USB HDDs become compatible?
Because as per Synology compatibility list only selected USB Storage Devices are compatible.

**Reply by 007revad** [reply, marked as answer]
Without the script all USB HDDs are already compatible.

**Reply by wolffcatskyy** [reply]
> Without the script all USB HDDs are already compatible.

Yea BUT with your script we could have USB RAID!!!  ;)
